### ISSUE TITLE
What happens in iOS or WinApp

### ISSUE THREAD

**Issue by megajogos** [airsdk/ANE-PlayAssetDelivery#10]
What happens with the iOS app or winApp? Our game is multiplatform we have to do a diferent code for iOS and Android?

**Comment by ajwfrost** [comment]
Yes, this is just for Android (well, only for apps that are distributed via the Google Play Store). For other OS - you could perhaps write an abstraction layer to be able to access assets regardless of which OS/mechanism is used, but you'd need to use the 'install-time' mechanism on Android I guess. Unless you also have async download of asset files on the other platforms...
